index.html

```diff
@@ -1,26 +1,35 @@
-jessica Baker
-
-"About"
-My name is Jessica Baker. I am 30 years old. I live in Palatine Il
-with my 3 chrildren. I am a student at CTD school.My favorite subject
-is math. My hobbies are listening to music and taking my chrildren 
-out to fun places.
-
-"Experience"
--javascript
--linux
-
-"Connect"
-https://github.com/Jessica12492/intro-to-programming-dorado
-https://mail.google.com
-https://www.facebook.com
-
-
-
-
-
-
-
-
-
-
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Personal Portfolio Website</title>
+  <meta charset="UTF-8">
+</head>
+
+<body>
+  <section>Jessica Baker
+    <h2>About</h2>
+    <p> name is Jessica Baker. I am 30 years old. I live in Palatine il with my 3
+      children. I am a student at CTD school. My favorite subject is math. My hobbies are listening to music , shopping
+      and taking my children out to fun places.
+  </section>
+  </p>
+  <section>
+    <h2>Experience</h2>
+    <ul>
+      <li>javascript</li>
+      <li>linux </li>
+    </ul>
+  </section>
+  <section>
+    <h3>Connect</h3>
+    <ul>
+      <li> https://github.com/Jessica12492/intro-to-programming-dorado</li>
+      <li> https://mail.google.com</li>
+      <li> https://www.facebook.com</li>
+    </ul>
+  </section>
+  <img src="../myself.jpg" alt="Picture of Me" width="100" height="90" />
```

Comment:
@Jessica12492  Nice, you've added the picture to the repo 🎉 ; last thing is to just change this `src` to `./myself.jpg` (one dot) instead of `..myself.jpg` (two dots) so that the code knows that the picture is in the **same folder** rather than in the **folder outside of intro-to-programming-dorado**